builtin/providers/influxdb/resource_user.go

```diff
@@ -0,0 +1,259 @@
+package influxdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/influxdata/influxdb/client"
+)
+
+func resourceUser() *schema.Resource {
+	return &schema.Resource{
+		Create: createUser,
+		Read:   readUser,
+		Update: updateUser,
+		Delete: deleteUser,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"password": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"admin": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
There doesn't seem to be support for an admin update. What happens if a user is an admin and then no longer is an admin?